messagebus/src/main/java/com/yahoo/messagebus/Sequencer.java

```diff
@@ -148,7 +148,7 @@ private class SequencedSendTask implements Messenger.Task {
         private final Message msg;
         SequencedSendTask(Message msg) { this.msg = msg; }
         @Override public void run() { sequencedSend(msg); }
-        @Override public void destroy() { msg.discard(); }
+        @Override public void destroy() { }
```

Comment:
This is the fix.

Reply:
`ResenderTask` does the same. I find it a bit weird that all tasks are destroyed. It would make more sense to me to just destroy anything left in the messenger queue upon _its_ destruction ...

Reply:
(Also, that wouldn't generate any replies, violating the claim I read somewhere that "message has guaranteed reply". This clearly isn't true 😂)

Reply:
I wonder if we should remember if run has been called, and call discard if not.

Reply:
Yes, I wondered about that, too. I'm not too fond of worshipping false gods, but perhaps this one is real, and the sacrifice seems minor.